### ISSUE TITLE
In LocalPayment workflow , How can I get paymentId in the latest SDK 5.2.0?

### ISSUE THREAD

**Issue by Jake-Jiang** [braintree/braintree_android#1207]
### Is your feature request related to a problem? Please describe.

LocalPaymentAuthRequestParams. paymentId can only be accessed from within the same library group (referenced groupId=com. braintreepayments. api

**Comment by warmkesselj** [comment]
Hey @Jake-Jiang this is a private property. I'm curious what you'd be needing the paymentId for?

**Comment by Jake-Jiang** [comment]
On sdk 4.x, We can get the paymentId  easily.
`localPaymentClient.startPayment(request, (result, error) -> {
      // do any preprocessing result.getPaymentId()
      localPaymentClient.approvePayment(MyActivity.this, result);
    });
  }`

we need send  paymentId  to our server.

According to your official document :https://developer.paypal.com/braintree/docs/guides/local-payment-methods/server-side/ruby#local-payment-method-webhooks

Braintree recommends that you store the payment_id returned to your client when starting the payment, and map it to a shopping cart identifier or some other form of identifier. When this webhook is received by your server, you can then look up the pending order using the included payment_id.

I see the latest  iOS SDK also  can get the   paymentId 

`// MARK: - BTLocalPaymentRequestDelegate Conformance

extension IdealViewController: BTLocalPaymentRequestDelegate {

    func localPaymentStarted(
        _ request: BraintreeLocalPayment.BTLocalPaymentRequest,
        paymentID: String,
        start: @escaping () -> Void
    ) {
        paymentIDLabel.text = "LocalPayment ID: \(paymentID)"
        start()
    }
}`

**This issue block  our sdk upgrade plan.**

**Comment by warmkesselj** [comment]
@Jake-Jiang can you try inspecting `LocalPaymentAuthRequest.ReadyToLaunch` in `LocalPaymentFragment.java` in the demo app. 

((LocalPaymentAuthRequest.ReadyToLaunch) paymentAuthRequest).requestParams.paymentId

```
localPaymentClient.createPaymentAuthRequest(request, (paymentAuthRequest) -> {
            if (paymentAuthRequest instanceof LocalPaymentAuthRequest.ReadyToLaunch) {
                LocalPaymentPendingRequest pendingRequest = localPaymentLauncher.launch(requireActivity(),
                ...
```

**Comment by warmkesselj** [comment]
@Jake-Jiang we have created a story for this issue and will follow up with it.

**Comment by sarahkoop** [comment]
Hey @Jake-Jiang - this has been released in `5.3.0`!
